v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -414,10 +414,11 @@ export const GraphDataConfigurationModel = DataConfigurationModel
     })
   }))
   .views(self => ({
-    casesInRange(min: number, max: number, attrId: string, cellKey: Record<string, string>) {
+    casesInRange(min: number, max: number, attrId: string, cellKey: Record<string, string>, inclusiveMax = true) {
       return self.subPlotCases(cellKey)?.filter(caseId => {
         const caseValue = self.dataset?.getNumeric(caseId, attrId)
-        if (isFiniteNumber(caseValue) && caseValue >= min && caseValue <= max) {
+        const upperRangeBoundary = inclusiveMax ? max : max - 0.000001
```

Comment:
These sorts of hard-coded offsets always make me nervous. Seems like it would be better as a percentage, e.g. max * 0.999999 because then at least it would be less resistant to magnitude effects.